src/Illuminate/Collections/LazyCollection.php

```diff
@@ -792,6 +792,17 @@ public function mergeRecursive($items)
         return $this->passthru('mergeRecursive', func_get_args());
     }
 
+    /**
+     * Merge the given items with the collection.
+     *
+     * @param  \Illuminate\Contracts\Support\Arrayable<TKey, TValue>|iterable<TKey, TValue>  $items
+     * @return static
+     */
+    public function mergeUnshift($items)
```

Comment:
Can you add tests to types here?

https://github.com/laravel/framework/blob/9.x/types/Support/LazyCollection.php